.circleci/config.yml

```diff
@@ -414,14 +414,14 @@ jobs:
             cd monitor-ci;
             echo "${CIRCLE_BRANCH} :: ${HERCULES_TOKEN} :: ${CIRCLE_PULL_REQUEST}";
```

Comment:
Is this echo still useful?  Since the other one is cleaned up in this PR, it seems like an ideal time to get rid of this one if there's no real reason to have it around.

Reply:
it was for debugging. nice catch.